packages/client/src/pages/index.tsx

```diff
@@ -7,15 +8,25 @@ import {
   ClientSettingService,
   useClientSettingState
 } from '@xrengine/client-core/src/admin/services/Setting/ClientSettingService'
+import WarningMessage from '@xrengine/client-core/src/common/components/WarningMessage/WarningMessage'
 import ProfileMenu from '@xrengine/client-core/src/user/components/UserMenu/menus/ProfileMenu'
 
 const ROOT_REDIRECT: any = globalThis.process.env['VITE_ROOT_REDIRECT']
 
 export const HomePage = (): any => {
   const { t } = useTranslation()
   const clientSettingState = useClientSettingState()
+  const [isHardwareAccelerationEnabled, setIsHardwareAccelerationEnabled] = useState(true)
   const [clientSetting] = clientSettingState?.client?.value || []
 
+  useEffect(() => {
```

Comment:
we want to put this in the location page, not the home page. 
https://github.com/XRFoundation/XREngine/tree/dev/packages/client-core/src/components/World/LoadEngineWithScene.tsx

Reply:
Basically, I added the check on the home page because I thought that having hardware acceleration enabled will be also important when the user navigates to the editor as well, then I preferred to add a check on the home page instead of adding in both location and editor. what do you think?